prog/dftb+/lib_type/typegeometry.F90

```diff
@@ -33,8 +33,8 @@ module dftbp_typegeometry
     !> atomic species
     integer, allocatable :: species(:)
 
-    !> coordinates of atoms
-    real(dp), allocatable :: coords(:,:)
+    !> coordinates of atoms, potentially in multiple replicas
+    real(dp), allocatable :: coords(:,:,:)
```

Comment:
This is probably breaking any access to the coords array in the code, needs to be very carefully checked.
See: https://github.com/dftbplus/dftbplus/pull/520/checks?check_run_id=770056167#step:6:18102